README.adoc

```diff
@@ -30,22 +59,22 @@ The canonical way to make use of `Pauser` is below:
     }
 ----
 
-The various different implementations of `Pauser` allow for varied pausing strategies - see
-http://openhft.github.io/Chronicle-Threads/apidocs/index.html[javadoc]
+The various implementations of `Pauser` allow for varied pausing strategies - see the
+http://openhft.github.io/Chronicle-Threads/apidocs/index.html[javadoc].
 
-== Pauser modes
+=== Pauser modes
 
-For the best performance, the default `busy` `PauserMode` will minimise jitter however it will maximise CPU usage.
-If you have too many threads in `busy` mode you can have a slow machine.
+For the best performance, the default `busy` Pauser mode minimises jitter. However, it does maximise CPU usage. If there are too many threads in `busy` mode, a machine may slow down.
```

Comment:
it does maximise CPU usage and your CPU's will run hotter.

Reply:
Added.